src/Bicep.Core/Analyzers/Linter/Rules/NoUnusedParametersRule.cs

```diff
@@ -12,6 +12,7 @@ namespace Bicep.Core.Analyzers.Linter.Rules
 {
     public sealed class NoUnusedParametersRule : NoUnusedRuleBase
     {
+        private const string MissingName = "<missing>";
```

Comment:
Same as the other rule - this is an implementation detail, please avoid doing it this way.

Reply:
@bhsubra I have fixed the thing @StephenWeatherford wanted me to fix.
I do not understand how the failing test is connected to my changes (and it only fails in some environments)